components/Header.jsx

```diff
@@ -1,11 +1,18 @@
 import AccountInfo from "@/components/AccountInfo";
 import styles from "@/styles/components/header.module.scss";
+import Link from "next/link";
+import { useRouter } from "next/router";
 
 export default function Header() {
+	const router = useRouter();
+	const homeLink =
+		router.asPath === "/" ? undefined : <a href='/'>HOME</a>; /* if homepage homeLink will be undefined so it wont be rendered */
```

Comment:
Comments, for most of the time, are for clarifying code's purpose that is otherwise hard to undersand
and sometimes can be used for orgenizing stuff.
But neither is the case here as the code explains itself, so remove it.

*Good code strives to be self explainatory and clear.

Reply:
noted